detection-rules/impersonation_chase.yml

```
@@ -37,6 +37,11 @@ source: |
         )
       )
     )
+    or strings.icontains(body.current_thread.text, 'Chase Privacy Operations')
+  )
+  and not (
+    sender.display_name is not null
+    and sender.display_name in~ ("chaser", "case")
   )
   and sender.display_name not in~ ("chaser", "case")
   and sender.email.domain.root_domain not in~ ('chase.com', 'united.com', 'transunion.com', 'shopping-chase.com')
```

The logic you added is perfect.  When considering any negations, we need to consider if the value can be null, so good work spotting that. But we should be able to collapse it into the existing condition without creating another one. 

```suggestion
    or strings.icontains(body.current_thread.text, 'Chase Privacy Operations')
  )
  and not (
    sender.display_name is not null and sender.display_name in~ ("chaser", "case")
  )
  and sender.email.domain.root_domain not in~ ('chase.com', 'united.com', 'transunion.com', 'shopping-chase.com')
```